packages/next/src/next-devtools/dev-overlay/components/devtools-panel/devtools-panel-tab/issues-tab/issues-tab-sidebar.tsx

```diff
@@ -0,0 +1,154 @@
+import type { ReadyRuntimeError } from '../../../../utils/get-error-by-type'
+
+import { Suspense, useMemo } from 'react'
+
+import { css } from '../../../../utils/css'
+import { getFrameSource } from '../../../../../shared/stack-frame'
+import { useFrames } from '../../../../utils/get-error-by-type'
+import { getErrorTypeLabel } from '../../../../container/errors'
+import { IssuesTabSidebarFrameSkeleton } from './issues-tab-sidebar-frame-skeleton'
+
+export function IssuesTabSidebar({
+  runtimeErrors,
+  activeIdx,
+  setActiveIndex,
+}: {
+  runtimeErrors: ReadyRuntimeError[]
+  errorType: string | null
+  activeIdx: number
+  setActiveIndex: (idx: number) => void
+}) {
+  return (
+    <aside data-nextjs-devtools-panel-tab-issues-sidebar>
+      {runtimeErrors.map((runtimeError, idx) => {
+        const isActive = idx === activeIdx
+        return (
+          <Suspense
+            key={idx}
+            fallback={<IssuesTabSidebarFrameSkeleton isActive={isActive} />}
+          >
+            <IssuesTabSidebarFrame
+              runtimeError={runtimeError}
+              idx={idx}
+              isActive={isActive}
+              setActiveIndex={setActiveIndex}
+            />
+          </Suspense>
```

Comment:
We should not delay navigation items. Instead, these items should be immediately visible and we should suspend lower on just the path with a proper fallback. So the Suspense needs to be moved inside `IssuesTabSidebarFrame` and just around the thing that suspends.